ledger/daml-on-sql/README.rst

```diff
@@ -1,937 +0,0 @@
-.. Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
Much of this document is quite general and not documented anywhere else. I believe we need to move large parts of it to the general documentation.